### PR TITLE
#211: further improving automatic checking

### PR DESCRIPTION
**Pull request by commjoen**
WIP for #211

### CODE REVIEW

**Comment by OMTGreleaser** [comment]
Results for commit 914b8864bd020e40306c484ecc5fafe2a43c1778: Broken link result: 0 .
  
 Markdown result errorlength: 0 .

**Comment by OMTGreleaser** [comment]
Results for commit 610fde5189ac63753bed5d79203438b955bd673c: Broken link result: 0 .
  
 Markdown result errorlength: 0 .

**Comment by OMTGreleaser** [comment]
Results for commit 74d3d49a080fe3c4df8d6b6b3f66684672cf1957: Broken link result: 0 .
  
 Markdown result errorlength: 0 .

**Comment by OMTGreleaser** [comment]
Results for commit f69b9d0b32a4e92fad0c949de88477f58a615f0a: Broken link result: 0 .
  
 Markdown result errorlength: 0 .

File: Document-fr/0x02-Frontispiece.md [hunk 1]
```diff
@@ -6,9 +6,9 @@
...
 
-Le MASVS est l'aboutissement d'un effort communautaire et de retours d'expérience venant du monde de l'industrie. Le but de ce standard est d'évoluer dans le temps et, dans ce cadre, tout retour est le bienvenu. Le meilleur moyen de rentrer en contact avec nous est via le canal Slack OWASP Mobile Project: https://owasp.slack.com/messages/project-mobile_omtg/details/
+Le MASVS est l'aboutissement d'un effort communautaire et de retours d'expérience venant du monde de l'industrie. Le but de ce standard est d'évoluer dans le temps et, dans ce cadre, tout retour est le bienvenu. Le meilleur moyen de rentrer en contact avec nous est via le canal Slack OWASP Mobile Project: <https://owasp.slack.com/messages/project-mobile_omtg/details/>
 
-Un compte peut être créé à l'endroit suivant: http://owasp.herokuapp.com/.
+Un compte peut être créé à l'endroit suivant: <http://owasp.herokuapp.com/.>
```
Comment (sushi2k) [inline Document-fr/0x02-Frontispiece.md]: change .> to >.

**Comment by OMTGreleaser** [comment]
Results for commit 3446577dd32fc7a87279caafad1b2c8df77f28c0: Broken link result: 0 .
  
 Markdown result errorlength: 0 .

**Pull request merged**